### PR TITLE
Docs: Edit of README and other topics in Contribute (part 1 of doc quality project)

### PR DESCRIPTION
**Pull request by josmperez**
Doc quality improvements.

### CODE REVIEW

File: contribute/merge-pull-request.md [hunk 1]
```diff
@@ -30,74 +30,75 @@ The pull request title should be formatted according to `<Area>: <Summary>` (Bot
...
 
 - Include any bug fix in general.
 - Include any change that you think would be interesting for the community as a whole.
-- Skip larger features divided in multiple pull requests since they might go into the release blog post/What's New article.
-- Use your best judgement and/or ask other maintainers for advice.
-- Including a change in error rather than skipping one that should have been there is better.
-- Always keep [Format the pull request title](#format-the-pull-request-title) in mind.
+- Skip larger features divided into multiple pull requests since they might go into the release blog post (What's New article).
+- Use your best judgement and, if you aren't sure, then ask other maintainers for advice.
+- It's better to include a change in error rather than skip one that should have been there. Err on the side of inclusion.
```
Comment (Eve832) [inline contribute/merge-pull-request.md:67]: this sentence is kind of difficult to parse. perhaps this is better: 
"It's better to err on the side of inclusion. Introducing an unnecessary or duplicate change in the changelog is better than leaving out a change."

File: contribute/merge-pull-request.md [hunk 2]
```diff
@@ -30,74 +30,75 @@ The pull request title should be formatted according to `<Area>: <Summary>` (Bot
...
-#### Include in changelog/release notes
+#### Include in changelog
 
-To include a pull request in the changelog/release notes you need to add a label named `add to changelog` to the pull request. Then additional validation rules is checked:
+To include a pull request in the changelog (release notes) you need to add a label named `add to changelog` to the pull request. Then the following additional validation rules are checked:
 
-- Title need to be formatted according to [Format the pull request title](#format-the-pull-request-title)
-- Description needs to include a breaking change notice if change is labeled to be a breaking change, see Breaking changes below for more information.
+- Title must be formatted according to [Format the pull request title](#format-the-pull-request-title)
+- Description must include a breaking change notice if the change is labeled to be a breaking change. Refer to [Breaking changes](#breaking-changes) below for more information.
```
Comment (Eve832) [inline contribute/merge-pull-request.md:81]:
```suggestion
- The title must be formatted according to [Format the pull request title](#format-the-pull-request-title)
- The description must include a breaking change notice if the change is labeled to be a breaking change. Refer to [Breaking changes](#breaking-changes) below for more information.
```

File: contribute/merge-pull-request.md [hunk 3]
```diff
@@ -118,39 +119,44 @@ In case the pull request introduces a breaking change you should document this.
...
+- `product-approved` label: Urgent fixes which need product approval, in order to get merged
+- `type/docs` label: Docs changes
+- `type/ci` label: Changes to the CI automation
 
 > **Note:** You can still backport a pull request after it's been merged.
 
 ## Doing the actual merge
 
-Time to actually merge the pull request changes. All commits in a pull request are squashed, hence the GitHub `Squash and merge` button is used to initialize the merge.
+The best time to actually merge the pull request varies from case to case. All commits in a pull request are squashed (hence, we use the GitHub `Squash and merge` button to initialize the merge).
```
Comment (Eve832) [inline contribute/merge-pull-request.md:142]:
```suggestion
The best time to actually merge the pull request varies from case to case. All commits in a pull request are squashed.
```

File: contribute/merge-pull-request.md [hunk 4]
```diff
@@ -118,39 +119,44 @@ In case the pull request introduces a breaking change you should document this.
...
+
+The merging process presents you with options for optionally change the commit message before merging. Please remember that developers might use the commit information when reviewing changes of files, doing Git blame, resolving merge conflicts, and so on.
```
Comment (Eve832) [inline contribute/merge-pull-request.md:144]:
```suggestion
You can change the commit message before merging. Please remember that developers might use the commit information for tasks like reviewing changes of files, doing Git blame, and resolving merge conflicts. 
```

File: contribute/merge-pull-request.md [hunk 5]
```diff
@@ -118,39 +119,44 @@ In case the pull request introduces a breaking change you should document this.
...
 
 ## Doing the actual merge
 
-Time to actually merge the pull request changes. All commits in a pull request are squashed, hence the GitHub `Squash and merge` button is used to initialize the merge.
+The best time to actually merge the pull request varies from case to case. All commits in a pull request are squashed (hence, we use the GitHub `Squash and merge` button to initialize the merge).
+
+The merging process presents you with options for optionally change the commit message before merging. Please remember that developers might use the commit information when reviewing changes of files, doing Git blame, resolving merge conflicts, and so on. 
 
-This will present you with options allowing you to optionally change the commit message before merging. Please remember that developers might use the commit information when reviewing changes of files, doing git blame and resolving merge conflicts etc., trying to quickly figure out what the actual change was. But there's not really any best practices around this, the following is an attempt to bring some guidance.
+Arguably, there's not really any best practices around this process, but consider the following items:
```
Comment (Eve832) [inline contribute/merge-pull-request.md:146]:
```suggestion
While there aren't formal best practices around this process, you can consider the following guidance:
```

File: contribute/merge-pull-request.md [hunk 6]
```diff
@@ -118,39 +119,44 @@ In case the pull request introduces a breaking change you should document this.
...
 - Remove any commit information that doesn't bring any context to the change.
 
-Consider:
+**Consider:**
 
-- Add any references to issues that the pull request fixes/closes/references to ease giving quick context to things. Doing this allows cross-reference between the commit and referenced issue(s).
+- Keep any references to issues that the pull request fixes, closes, or references. Doing this allows cross-reference between the commit and referenced issue or issues.
 
-Finalize the merge by clicking on the `Confirm squash and merge` button.
+Finalize the merge by clicking the `Confirm squash and merge` button.
```
Comment (Eve832) [inline contribute/merge-pull-request.md:158]: not sure why we're putting UI button text in monocode rather than bold?

Comment (Eve832) [inline contribute/merge-pull-request.md:158]:
```suggestion
Finalize the merge by clicking **Confirm squash and merge**.
```

File: contribute/triage-issues.md [hunk 7]
```diff
@@ -29,4 +29,4 @@ Read more about [how we prioritize issues](/ISSUE_TRIAGE.md#4-prioritization-of-
 
 ## Report duplicates
 
-If you find two issues that describe the same thing, add a comment in one of the issues, with a reference (`#<issue number>`) to the other. Explain why you think the issue is duplicated.
+If you find two issues that describe the same thing, add a comment in one of the issues. In the comment, include a reference (`#<issue number>`) to the other. Explain why you think the issue is duplicated.
```
Comment (Eve832) [inline contribute/triage-issues.md:32]:
```suggestion
If you find two issues that describe the same thing, add a comment in one of the issues. In the comment, include a reference (`#<issue number>`) to the duplicate. Explain why you think the issue is duplicated.
```

**Review by Eve832 — CHANGES_REQUESTED** [review]

File: contribute/README.md [hunk 8]
```diff
@@ -1,18 +1,24 @@
...
 - [Triage issues](triage-issues.md)
 - [Merge a pull request](merge-pull-request.md)
 - [Making changes to the CI pipeline](drone-pipeline.md)
 - [Breaking changes in frontend APIs](./breaking-changes-guide/breaking-changes-guide.md)
 
-The `style-guides` directory contains style guides for the Grafana software project and documentation.
+Learn how to create great documentation and apps for Grafana:
+
+- [Contribute to our documentation](../contribute/documentation/README.md) - Learn how to write and style Grafana documentation.
+- [Developer portal] - Find resources for creating apps and plugins to Grafana.
```
[the remark this answers is not in the document]
Comment (josmperez) [reply contribute/README.md:17]: it does now

File: contribute/merge-pull-request.md [hunk 9]
```diff
@@ -30,74 +30,75 @@ The pull request title should be formatted according to `<Area>: <Summary>` (Bot
...
 **Example:**
+
 `Docs: Change url to URL in all documentation files`
 
 See [formatting guidelines](create-pull-request.md#formatting-guidelines) for more information.
 
 ### Assign a milestone
 
-Several things in the Grafana release process requires at least pull requests to be in a milestone, for example [generating changelog/release notes](#include-in-changelog-and-release-notes).
+The Grafana release process requires at least some pull requests to be in a milestone. For example, [generating changelog (release note)](#include-in-changelog-and-release-notes) must be in a milestone.
```
[the remark this answers is not in the document]
Comment (josmperez) [reply contribute/merge-pull-request.md]: yes. I have tightened up this whole section to make it more clear that this is an automated process

File: contribute/merge-pull-request.md [hunk 10]
```diff
@@ -30,74 +30,75 @@ The pull request title should be formatted according to `<Area>: <Summary>` (Bot
...
+- Pull requests targeting `main` should use the `.x` milestone of the next minor (or major) version (you can find that version number inside the `package.json` file).
+- Backport pull requests should use the version of the target branch (for example, `9.4.x` for the `v9.4.x` branch).
 
-### Include in changelog and release notes?
+### What to include in changelog and release notes
 
-At Grafana we generate the [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md) and [release notes](https://grafana.com/docs/grafana/latest/release-notes/) based on merged pull requests. Including changes in the changelog/release notes is very important to provide a somewhat complete picture of what changes a Grafana release actually includes.
+At Grafana we generate the [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md) and [release notes](https://grafana.com/docs/grafana/latest/release-notes/) based on merged pull requests. Including changes in the changelog (release notes) is very important to provide a somewhat complete picture of what changes a Grafana release actually includes.
```
[the remark this answers is not in the document]
Comment (josmperez) [reply contribute/merge-pull-request.md]: changed to "relatively complete"

File: contribute/merge-pull-request.md [hunk 11]
```diff
@@ -30,74 +30,75 @@ The pull request title should be formatted according to `<Area>: <Summary>` (Bot
...
 
 Exactly what changes should be added to the changelog is hard to answer but here's some general guidance:
 
 - Include any bug fix in general.
 - Include any change that you think would be interesting for the community as a whole.
-- Skip larger features divided in multiple pull requests since they might go into the release blog post/What's New article.
-- Use your best judgement and/or ask other maintainers for advice.
-- Including a change in error rather than skipping one that should have been there is better.
-- Always keep [Format the pull request title](#format-the-pull-request-title) in mind.
+- Skip larger features divided into multiple pull requests since they might go into the release blog post (What's New article).
```
[the remark this answers is not in the document]
Comment (josmperez) [reply contribute/merge-pull-request.md:63]: fixed

File: contribute/merge-pull-request.md (hunk 1 above)
Comment (josmperez) [reply contribute/merge-pull-request.md:65]: nice

File: contribute/create-pull-request.md [hunk 12]
```diff
@@ -42,76 +42,76 @@ Pull requests for Redux contributions must:
...
 - **Betterer also runs as part of our CI**:
-  - You may see the following error message in the CI: `Unexpected changes detected in these tests while running in CI mode`. To resolve, merge with the target branch (usually `main`).
-  - You may see merge conflicts for the `.betterer.results` file. To resolve, merge with the target branch (usually `main`) then run `yarn betterer:merge` and commit.
+  - You may see the following error message in the CI: `Unexpected changes detected in these tests while running in CI mode`. To resolve the error, merge with the target branch (usually `main`).
+  - You may see merge conflicts for the `.betterer.results` file. To resolve, merge with the target branch (usually `main`), and then run `yarn betterer:merge` and commit.
 
-## Backend-specific guidelines
+## Guidelines for backend development
 
 Please refer to the [backend style guidelines](/contribute/backend/style-guide.md).
```
Comment (josmperez) [inline contribute/create-pull-request.md:76]:
```suggestion
Refer to the [backend style guidelines](/contribute/backend/style-guide.md).
```

**Review by Eve832 — APPROVED** [review]

**Pull request merged**